build-system/compile/post-closure-babel.js

```diff
@@ -35,19 +31,6 @@ async function terserMinify(code, filename) {
     },
     sourceMap: true,
   };
-  const basename = path.basename(filename, argv.esm ? '.mjs' : '.js');
-  if (!mainBundles) {
-    mainBundles = Object.keys(jsBundles).map((key) => {
-      const bundle = jsBundles[key];
-      if (bundle.options && bundle.options.minifiedName) {
-        return path.basename(bundle.options.minifiedName, '.js');
-      }
-      return path.basename(key, '.js');
-    });
-  }
-  if (mainBundles.includes(basename)) {
-    options.output.preamble = ';';
-  }
```

Comment:
Glad to see this (hacky?) block go away.

Reply:
Hopefully gone for good this time! 🤞